src/main/java/com/twilio/oai/template/AbstractApiActionTemplate.java

```diff
@@ -1,10 +1,12 @@
 package com.twilio.oai.template;
 
-import org.openapitools.codegen.CodegenConfig;
-
+import java.io.File;
 import java.util.List;
 import java.util.Map;
 
+import org.openapitools.codegen.CodegenConfig;
+import org.openapitools.codegen.SupportingFile;
+
 public abstract class AbstractApiActionTemplate implements IApiActionTemplate {
```

Comment:
Will adding one of these for Python fix the folder naming issue (DI-2448)"

Reply:
Don't think so. We'e need to override this method: https://github.com/OpenAPITools/openapi-generator/blob/081a6ef46699fb4cc954b663781f3374e9300153/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java#L2592-L2595